app/services/fact_parser.rb

```diff
@@ -52,6 +52,14 @@ def ipmi_interface
     raise NotImplementedError, not_implemented_error(__method__)
   end
 
+  def comment
+    facts[:foreman_comment]
+  end
+
+  def has_comment?
+    facts.has_key?(:foreman_comment)
```

Comment:
this is `key?` instead of `has_key?` now, I think, rubocop will probably yell on new run